src/Tickets/Admin/Tickets/List_Table.php

```diff
@@ -377,7 +390,18 @@ public function column_name( $item ): string {
 			'ticket_link' => $edit_post_link,
 		];
 
-		return $template->template( 'tickets/column/name', $context, false );
+		$name = $template->template( 'tickets/column/name', $context, false );
+
+		/**
+		 * Filters the name for the All Tickets Table.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $name The name for the All Tickets Table.
```

Comment:
Missing the `$item` as a param. I noticed in a few other filters below you are also missing the `$item` param.